src/sendSlackMessage.js

```diff
@@ -14,38 +46,98 @@ export default async function sendSlackMessage ({
     throw new Error('channel is required!')
   }
 
-  if (!message) {
-    throw new Error('message is required!')
+  if (!message && !text) {
+    throw new Error('message || token is required!')
   }
 
+  if (colorCodes[color]) {
+    color = colorCodes[color]
+  }
+
+  const colored = color?.match(/^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$/)
+
   debug = debug === 'true' || debug === true
 
   if (debug) { console.log(`token.length: ${token.length}, channel: ${channel}, message: ${message}`) }
 
-  const { WebClient } = await import('@slack/web-api')
   const { markdownToBlocks } = await import('@tryfabric/mack')
```

Comment:
can move this down to https://github.com/brave/security-action/pull/582/files#diff-febfc9f82835dd1630e1e3499112f32dca4efa0a57646fd2b11483784056667eR90 if you are trying to lazy-import stuff